recipes/ckanweb-deploy-exts.rb

```diff
@@ -292,6 +292,19 @@
 				source "supervisor-ckan-archiver.conf"
 				mode "0744"
 			end
+
+			template "/usr/local/bin/archiverTriggerAll.sh" do
+				source 'archiverTriggerAll.sh'
+				owner 'root'
+				group 'root'
+				mode '0755'
+			end
+
+			#Trigger at 10pm monday nights weekly
```

Comment:
Wouldn't the weekend be better?

Reply:
is after peak hour and url checks should be done by early morning. Another reason is do you want to be working on the weekend if it breaks?